s4s/src/main/kotlin/examples/Benchmarks.kt

```diff
@@ -0,0 +1,47 @@
+import bottomup.BottomUp
+import sketchral.InputFactory
+import sketchral.OutputParser
+import sketchral.Result
+import sketchral.withNegEx
+import util.*
+import java.io.File
+import java.io.IOException
+import java.util.concurrent.TimeUnit 
+
+data class TestQuery(val  functions:List<Func>, val names:List<String> , val query:Query){
```

Comment:
There aren't any examples of this constructor nor `runTest` being called, so I'm not totally sure if I have the right idea, but it seems to me like the values declared in files in `examples/` get passed to `TestQuery`. In that case, it would make a lot of sense to make some interface `I` containing the list of functions, names, and query, so `TestQuery` can take an `I` as input, and all the stuff in each `examples/` file can be wrapped in an object which implements `I`.
The syntax would be something like `object ListQuery implements I { ... }`

Reply:
Either is fine I'd rather get a MVP up so we're able to run tests and investigate the remaining questions